### PR TITLE
Tabular representation of time-based features

### PR DESCRIPTION
**Pull request by mholthausen**
This MR

- Adds a tabular representation of time-based features to `FeatureInfoGrid`

image

@terrestris/devs please review

### CODE REVIEW

File: src/component/FeatureInfoGrid/FeatureInfoGrid.tsx [hunk 1]
```diff
@@ -165,6 +169,38 @@ export class FeatureInfoGrid extends React.Component<FeatureInfoGridProps, Featu
...
+   */
+  getTimeFeatureColumnDefs(): any[] {
+    const {
+      selectedFeat
+    } = this.state;
+
+    const columnDefs: any[] = [];
+
+    Object.keys(selectedFeat.getProperties()).forEach(featureColumnKey => {
+      const prop = selectedFeat.getProperties()[featureColumnKey];
```
Comment (jansule) [inline src/component/FeatureInfoGrid/FeatureInfoGrid.tsx:184]:
```suggestion
      const prop = selectedFeat.get(featureColumnKey);
```
minor

File: src/component/FeatureInfoGrid/FeatureInfoGrid.tsx [hunk 2]
```diff
@@ -190,6 +226,45 @@ export class FeatureInfoGrid extends React.Component<FeatureInfoGridProps, Featu
...
+
+    const featureLayerName: string = feat.get('layerName');
+    const filterFeatures: any[] = features.filter(f => {
+      return f.get('layerName') === featureLayerName;
+    });
+
+    filterFeatures.forEach(filterFeature => {
+      const colData = {};
+      Object.keys(filterFeature.getProperties()).forEach(propKey => {
+        const prop = filterFeature.getProperties()[propKey];
```
Comment (jansule) [inline src/component/FeatureInfoGrid/FeatureInfoGrid.tsx:251]:
```suggestion
        const prop = filterFeature.get(propKey);
```
minor

File: src/component/FeatureInfoGrid/FeatureInfoGrid.tsx [hunk 3]
```diff
@@ -190,6 +226,45 @@ export class FeatureInfoGrid extends React.Component<FeatureInfoGridProps, Featu
...
+        if (
+          prop instanceof OlGeomGeometry ||
+          (typeof prop !== 'string' && typeof prop !== 'number')
+        ) {
+          return;
+        }
+        if (propKey === 'layerName') {
+          return;
+        }
+        colData[propKey] = filterFeature.getProperties()[propKey];
```
Comment (jansule) [inline src/component/FeatureInfoGrid/FeatureInfoGrid.tsx:261]:
```suggestion
        colData[propKey] = filterFeature.get(propKey);
```
minor

**Review by jansule — APPROVED** [review]
LGTM. Just some minor remarks

**Comment by mholthausen** [comment]
Thanks for your review!

**Pull request merged**